hw/dv/tools/dvsim/questa.hjson

```diff
@@ -0,0 +1,106 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  build_cmd:  "vlib work && {job_prefix} vlog"
+  run_cmd:    "{job_prefix} vsim"
+
+  // Indicate the tool specific helper sources - these are copied over to the
+  // {tool_srcs_dir} before running the simulation.
+  tool_srcs:  ["{dv_root}/tools/questa/*"]
+
+  build_opts: ["-sv",
+               "-timescale 1ns/1ps",
+               "+incdir+/${QUESTA_HOME}/verilog_src/uvm-1.2/src/",
+               "/${QUESTA_HOME}/verilog_src/uvm-1.2/src/uvm_pkg.sv",
+               "-mfcu"
+               "-f {sv_flist}"]
+
+  run_opts:   ["-sv_seed {seed}",
+               // List multiple tops for the simulation
+               "{sim_tops}",
+               "-suppress vsim-8323",
+               "-c",
+               "-sv_lib /${QUESTA_HOME}/uvm-1.2/linux_x86_64/uvm_dpi",
+               "-lib {sv_flist_gen_dir}/work",
+               "+show_file_path=1",
+               "-permit_unmatched_virtual_intf",
+               "+UVM_TESTNAME={uvm_test}",
+               "+UVM_TEST_SEQ={uvm_test_seq}",
+               '''-do \"run -all; exit\"'''
+               ]
+
+
+  // Supported wave dumping formats (in order of preference).
+  supported_wave_formats: []
+
+  // Default tcl script used when running the sim. Override if needed.
+  run_script: "{tool_srcs_dir}/questa_run.do"
```

Comment:
With #4538, everything is now referenced from `proj_root`.

Reply:
Note that `hw/dv/tools/sim.tcl` is designed to be a tool-common tcl script that works with VCS and Xcelium. Perhaps that works for Questa as well? It already supports wave dumping with common formats.

Reply:
Let me look at this next PR